lib/efi_event.h

```diff
@@ -187,4 +187,19 @@ typedef struct {
     // UINT8 OptionalData[];
 } PACKED EFI_LOAD_OPTION;
 
+/*
+ * EV_NO_ACTION_STRUCT is the structure of an EV_NO_ACTION event.
+ * Described in TCG PCClient FPF section 9.4.5.
```

Comment:
Oh, ugh, it's PFP not FPF.  https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PFP_r1p05_05_3feb20.pdf

Reply:
@MikeSpreitzer, please can you then fix the typo :)

Reply:
fixed